specification/deviceprovisioningservices/resource-manager/Microsoft.Devices/stable/2022-12-12/iotdps.json

```diff
@@ -0,0 +1,2571 @@
+{
```

Comment:
There is one more PR, with a different new API version than this: [https://github.com/Azure/azure-rest-api-specs/pull/21736](https://github.com/Azure/azure-rest-api-specs/pull/21736). That also similar changes, which one to review?

Reply:
Yes, please ignore the #21736 one (closed). 

Also for context. The changes in this API version have already been approved in a private preview version. But we were not able to publish the changes to the public repo. As a result we were asked (by OpenAPI support) to create another PR manually in public repo. 

Ask: can we leverage that previous approval to expedite review on this pull request? We had gone through breaking change review as well. 

Please advise. 

https://github.com/Azure/azure-rest-api-specs-pr/pull/8870

Reply:
Any update on the above? @visingla-ms 
We would like to move along with this since it is a dependency for two features launch for us. Thank you!